lte/gateway/deploy/agw_install_docker.sh

```diff
@@ -14,7 +14,7 @@ MODE=$1
 RERUN=0    # Set to 1 to skip network configuration and run ansible playbook only
 WHOAMI=$(whoami)
 MAGMA_USER="ubuntu"
-MAGMA_VERSION="${MAGMA_VERSION:-master}"
+MAGMA_VERSION="${MAGMA_VERSION:-v1.8}"
```

Comment:
That version should also be used to modify `/opt/magma/lte/gateway/docker/.env`, since it is hardcoded there. 
* `master` -> `IMAGE_VERSION=latest`
* `v1.8` -> `IMAGE_VERSION=1.8.0`

Reply:
Good point, I will add that in the script.